doc/spec.adoc

```diff
@@ -76,6 +77,8 @@ A channel is composed of several fields:
 ** `maven-latest` - a version marked as `latest` in the Maven metadata
 ** `maven-release` - a version marked as `release` in the Maven metadata
 ** `none` - do not attempt to resolve versions of artifact not listed in the `streams` collection. Default value if no strategy is provided.
+* Optional `gpg-check` boolean indicating if the artifacts resolved from this channel have to have a valid GPG signature.
+* Optional `gpg-key` a list of URLs that the public GPG keys used to validate artifact signatures are resolved from.
```

Comment:
s/gpg-key/gpg-keys

```Optional `gpg-keys` a list of URLs corresponding to the public GPG keys used to validate artifact signatures.``` 

If `gpg-check` is true, `gpg-keys` must be set too, right?

Reply:
Not necessary. The client using wildfly-channels may have a local keystore that's used to resolve the required public keys, or they can use public keyservers. 

The `gpg-keys` field was meant to be an optional way to provide the channel definition with a way to pass information were the key may be found.